azure/durable_functions/models/actions/ContinueAsNewAction.py

```diff
@@ -13,7 +15,7 @@ class ContinueAsNewAction(Action):
     """
 
     def __init__(self, input_=None):
-        self.input_ = input_
+        self.input_ = dumps(input_, default=_serialize_custom_object)
```

Comment:
Should we write a test for this change?

Reply:
I did consider it! Considering that our serialization rules are still full of exceptions (the new work I'm doing in the extension should help simplify the serialization story) I'd rather wait until we patch our serialization rules.

It's also difficult to build confidence in these serialization patches as they can only be _truly_ tested when running on top of the Functions Host. So, for now, I think I'll punt that work item until we get back to our serialization work!

Reply:
Sounds good, that's why I asked, as I know this is currently fragile, so unit tests may not be the best use if we are changing this soon.